src/popup/pages/TransactionDetails.vue

```diff
@@ -226,29 +226,29 @@ export default defineComponent({
   setup(props, { root }) {
     const {
       setTransaction,
-      getTxSymbol,
-      getTxAmountTotal,
       isErrorTransaction,
       isAllowance,
       tokens,
       isDex,
-    } = useTransactionToken({
+    } = useTransaction({
       store: root.$store,
       showDetailedAllowanceInfo: true,
     });
 
     const transaction = ref<ITransaction>();
 
-    const getTx = computed(() => root.$store.getters.getTx);
-    const getTxTipUrl = computed(() => root.$store.getters.getTxTipUrl);
-    const getExplorerPath = computed(() => root.$store.getters.getExplorerPath);
-    const isTxAex9 = computed(() => root.$store.getters.isTxAex9);
+    const getTx = useGetter('getTx');
+    const getTxTipUrl = useGetter('getTxTipUrl');
+    const getExplorerPath = useGetter('getExplorerPath');
+    const isTxAex9 = useGetter('isTxAex9');
+    const getTxSymbol = useGetter('getTxSymbol');
+    const getTxAmountTotal = useGetter('getTxAmountTotal');
```

Comment:
I deliberately replaced those `useGetter`'s with computed props to satisfy tests. I think mocking up function on the tests side in order to have a little bit cleaner code on the FE side is not a good thing.